src/openMVG/dataio/ImageFeed.hpp

```diff
@@ -46,7 +46,7 @@ class ImageFeed : public IFeed
    * 
    * @see readCalibrationFromFile()
    */
-  ImageFeed(const std::string imagePath, const std::string calibPath);
+  ImageFeed(const std::string& imagePath, const std::string& calibPath);
```

Comment:
Update doc with the supported modes:
- direcotry
- sfm file
- txt file
- regex